packages/itwinui-css/src/slider/slider.scss

```diff
@@ -2,116 +2,174 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @use '../mixins';
 
-$thumb-height: var(--iui-size-m); // 16px
-$track-height: var(--iui-size-2xs); // 4px
-$tick-height: var(--iui-size-s); // 12px
+// #region Slider container
+.iui-slider-container {
+  --_iui-slider-container-gap: var(--iui-size-s);
```

Comment:
do we need all these variables or can we reduce some of them?

Reply:
I've removed the cursor CSS var.  I can replace some CSS vars with SASS vars, unsure if that would be preferred or not.

Reply:
CSS variables serve two main purposes:
- from inside: allow changing properties inside nested selectors without repeating those selectors.
- from outside: allow changing properties in inline styles. usually for the parts that need to change dynamically, such as the slider thumb position.

I think there is a readability concern with CSS variables, so it's better to avoid them if all you need is a place to store values. Sass variables are a perfectly fine alternative for storing values.

Reply:
I replaced a couple more CSS vars with SASS vars.

Reply:
`--_iui-slider-thumb-size` can also be replaced?

Reply:
Swapped that CSS var with a SASS var.